<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefs_base</class>
 <widget class="QWidget" name="prefs_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>151</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>qemu-img is the QEMU disk image utility</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel_kvm">
     <property name="statusTip">
      <string>KVM / QEMU Emulator</string>
     </property>
     <property name="text">
      <string>kvm / quemu</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel_vde_switch">
     <property name="toolTip">
      <string>Virtual Distributed Ethernet switch</string>
     </property>
     <property name="text">
      <string>vde_switch</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel_qemu_img">
     <property name="toolTip">
      <string>QEMU disk image utility</string>
     </property>
     <property name="text">
      <string>qemu-img</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="kurlrequester_exeKvm">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KUrlRequester" name="kurlrequester_exeVdeSwitch"/>
   </item>
   <item row="4" column="1">
    <widget class="KUrlRequester" name="kurlrequester_exeQemuImgCreator">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcolorbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
